CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# WIP for 3.3.0 release
```

Comment:
Can you make this 3.0.0 before merging? Thanks!

Reply:
Gah yes thanks!